serum-390/soen390-team16-godzilla
# Connecting Customer Management Frontend to Backend.

# Description
Can add the customer, edit customer information, delete customer (changes take place in the api).
The changes will be showed in the data grid.
Fix the duplicate "form" folder issue. Only one folder exists now.

A link to the work item E.g.: #125 #62 #60 

## How to Test
Click the "Add New Customer", "Edit", "Delete" button on customer management section.
See the changes in the data grid.
Go to "http://localhost:8080/api/salescontact/" to see the changes as well.
